src/views/CreatePage.vue

```diff
@@ -244,6 +256,9 @@ const uploadURI = async (uri: string, name: string) => {
           })
           await updateImageMetadata(fileRef)
 
+          // firebaseの更新終了
+          showDoneBtn()
+
           // store 更新
           if (typeof canvasVal === 'string') {
             await setCanvas(canvasVal)
```

Comment:
storeの更新まで終わってからdoneにした方がいいと思いました。
```suggestion
          // store 更新
          if (typeof canvasVal === 'string') {
            await setCanvas(canvasVal)
            
          // firebase, storeの更新終了
          showDoneBtn()
```